addon/components/ember-youtube.js

```diff
@@ -19,6 +19,8 @@ export default Ember.Component.extend({
 	startSeconds: undefined,
 	endSeconds: undefined,
 	suggestedQuality: undefined,
+        height: 360,
+        width: 270,
```

Comment:
Hi, glad you like it and yes — if you could adapt it to tabs it would be great. Thank you!

I've updated the `.editorconfig` accordingly: https://github.com/oskarrough/ember-youtube/commit/57e6e64e84bd2af86a3e04565bc217621cc19183